src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-
+gasPrice=1000
+gasLimit=70000
```

Comment:
Please structure the input configuration to have spring section, IO section, then the app specific parameters.